### ISSUE TITLE
ShouldNotHappenException: Trying to create a test without description.

### ISSUE THREAD

**Issue by mabdullahsari** [pestphp/pest#227]
- Issue: Trying to create a test without description.
- PHP version: 7.4.12
- Operating system: Darwin

---------

Happened after I included a trait in one of my enum classes and used the static function.
Don't know what else to give as context. The stack trace doesn't seem to give valuable information.

**UPDATE:**
Seems like it is caused by Spatie's [**laravel-enum**](https://github.com/spatie/enum/blob/master/src/Enum.php#L215) package. Trying to figure out why.

```
final class Unit extends Enumeration
{
    use HasLabels;

    public static function labels(): array
    {
        return static::translateValues(['METERS', 'PIECES', 'ROLLS']); // culprit
    }
}
```

**Comment by nunomaduro** [comment]
@mabdullahsari Let me know how your investigation goes.

**Comment by mabdullahsari** [comment]
> @mabdullahsari Let me know how your investigation goes.

The issue is interacting with the container. Even calling `Container::getInstance()` makes Pest crash. I'll take a thorough look when phpstorm 2020.3 is released with Xdebug 3 support. I've moved on with an alternative approach that doesn't make the testsuite crash. I'll keep you posted whenever I stumble across something.

**Comment by mabdullahsari** [comment]
@nunomaduro 

I tried this exact approach in another project which also uses Pest and the same thing happened over there as well. This probably means that Pest is trying to do some sort of preloading? (I don't know the internals, sorry). The `__` helper is run before the translator/container exists in the testsuite which is definitely the cause. Keep in mind that this only happens when using Pest. When running the application, everything is fine because the static labels function does not get called until the application itself has booted.

You might want to take a look at https://github.com/spatie/enum/blob/master/src/Enum.php#L215 as I don't really know where to hint you towards. L215 is the cause of the crash.

Also, see this stacktrace. I captured a debug_backtrace just before the application blows up due to the translator call.
https://pastebin.com/KhKYNMBG
